spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/configuration/ZeebeClientProdAutoConfiguration.java

```diff
@@ -0,0 +1,100 @@
+@AutoConfigureBefore(ZeebeClientAllAutoConfiguration.class)
+public class ZeebeClientProdAutoConfiguration {
+
+  private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  @Bean
+  public ZeebeClientConfiguration zeebeClientConfiguration() {
+    return new ZeebeClientConfiguration();
+  }
+
+  @Bean(destroyMethod = "close")
+  public ZeebeClient zeebeClient(final ZeebeClientConfiguration configuration) {
+    LOG.info("Creating ZeebeClient using ZeebeClientConfiguration [" + configuration + "]");
+    final ScheduledExecutorService jobWorkerExecutor = configuration.jobWorkerExecutor();
+    if (jobWorkerExecutor != null) {
+      final ManagedChannel managedChannel = ZeebeClientImpl.buildChannel(configuration);
+      final GatewayGrpc.GatewayStub gatewayStub =
+          ZeebeClientImpl.buildGatewayStub(managedChannel, configuration);
+      final ExecutorResource executorResource =
+          new ExecutorResource(jobWorkerExecutor, configuration.ownsJobWorkerExecutor());
+      return new ZeebeClientImpl(configuration, managedChannel, gatewayStub, executorResource);
+    } else {
+      return new ZeebeClientImpl(configuration);
+    }
+  }
+  // TODO: Interceptors
+  // TODO: applyOverrides()
+  /*
+  @Bean
+  public ZeebeClientBuilder builder(JsonMapper jsonMapper,
+                                    @Autowired(required = false) List<ClientInterceptor> clientInterceptorList) {
+    final ZeebeClientBuilderImpl builder = new ZeebeClientBuilderImpl();
+
+    builder.gatewayAddress(configurationProperties.getGatewayAddress());
+    builder.defaultJobPollInterval(configurationProperties.getDefaultJobPollInterval());
+    builder.defaultJobTimeout(configurationProperties.getDefaultJobTimeout());
+    builder.defaultJobWorkerMaxJobsActive(configurationProperties.getDefaultJobWorkerMaxJobsActive());
+    builder.defaultJobWorkerName(configurationProperties.getDefaultJobWorkerName());
+    builder.defaultMessageTimeToLive(configurationProperties.getDefaultMessageTimeToLive());
+    builder.numJobWorkerExecutionThreads(configurationProperties.getNumJobWorkerExecutionThreads());
+    builder.defaultRequestTimeout(configurationProperties.getDefaultRequestTimeout());
+    builder.credentialsProvider(configurationProperties.getCredentialsProvider());
+    builder.caCertificatePath(configurationProperties.getCaCertificatePath());
+    if (configurationProperties.isPlaintextConnectionEnabled()) {
+      builder.usePlaintext();
+    }
+    builder.withJsonMapper(jsonMapper);
+    final List<ClientInterceptor> legacyInterceptors = configurationProperties.getInterceptors();
+    if (!legacyInterceptors.isEmpty()) {
+      builder.withInterceptors(legacyInterceptors.toArray(new ClientInterceptor[0]));
+    } else if (clientInterceptorList != null && !clientInterceptorList.isEmpty()) {
+      builder.withInterceptors(clientInterceptorList.toArray(new ClientInterceptor[0]));
+    }
+    return builder;
+  }*/
```

Comment:
❌ 
```suggestion
```

Reply:
I've removed that before, not sure why it appeared again... Will fix, thank you